### ISSUE TITLE
Decode hash server side

### ISSUE THREAD

**Issue by mrcesc0** [Valve/fingerprintjs2#385]
Is there a way to decode the generated hash on the server side? Thank you

**Comment by GrosSacASac** [comment]
No, the point of the hash is that is not reversible in theory. https://en.wikipedia.org/wiki/Hash_function

**Comment by mrcesc0** [comment]
so, sorry for the ignorance! thank you
